raiden-ts/src/transport/epics.ts

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable @typescript-eslint/camelcase */
```

Comment:
Because of `user_id`? We should maybe add exceptions in our eslint config a la 
```
"@typescript-eslint/camelcase": ["error", { allow: ["user_Id"] }]
```
for variable names we don't have any control over

Reply:
I thought about it, but I'd rather not have that for the whole projects, as only some files need to deal with snake case names (mainly matrix and messages, as these communicate with the external world with functions and objects externally defined to be on snake case). Otherwise, I'd prefer to have camel enforced on the rest of the project.